tests/sph_image_test.py

```diff
@@ -21,7 +26,23 @@ def setup():
     f.gas['smooth']=(f.gas['mass']/f.gas['rho'])**(1,3)
     f.physical_units()
 
-def test_images():
+@pytest.fixture
```

Comment:
Note: in this file, the tests use fixtures (which are passed as arguments to the test functions). Note that I could also have updated the old versions to use the absolute path (`test_folder / "..."`) with the same end result.